README.md

```diff
@@ -7,58 +8,65 @@
 [![Slack](https://ilios-slack.herokuapp.com/badge.svg)](https://ilios-slack.herokuapp.com/)
 [![Stories in Ready](https://badge.waffle.io/ilios/frontend.png?label=ready&title=Ready)](https://waffle.io/ilios/frontend)
 
-## Web interface for accessing and managing [Ilios Platform](https://github.com/ilios/ilios) data.
+## Open Source Heroes
+
+Every day these for profit companies make developing Ilios possible.  Without the free tier of service they offer to
+open source software we would be nowhere so if you have a chance to pay them for something please do.
```

Comment:
"open source software, Ilios would be nowhere, so if you have a chance to pay them for something, please do!"